routers/web/repo/issue_stopwatch.go

```diff
@@ -22,7 +22,7 @@ func IssueStopwatch(c *context.Context) {
 
 	var showSuccessMessage bool
 
-	if !issues_model.StopwatchExists(c.Doer.ID, issue.ID) {
+	if !issues_model.StopwatchExists(c, c.Doer.ID, issue.ID) {
```

Comment:
I know, not your fault but still… :vomiting_face:
I can already foresee a follow-up PR.